business_objects/record.py

```diff
@@ -112,17 +157,45 @@ def get_missing_rats_records(
     return general.execute_all(query)
 
 
-def get_missing_tokenized_records(project_id: str, limit: int) -> List[Any]:
+def get_records_without_tokenization(
+    project_id: str, limit: int = 0, query_only: bool = False
+) -> List[Any]:
     query = f"""
-    SELECT r.id, r.data
+    SELECT r.id, r.data, rt."columns"
     FROM record r
     LEFT JOIN record_tokenized rt
-        ON r.id = rt.record_id 
-        AND r.project_id = rt.project_id 
+        ON r.id = rt.record_id
+        AND r.project_id = rt.project_id
         AND rt.project_id = '{project_id}'
-    WHERE r.project_id = '{project_id}' 
-    AND rt.id IS NULL    
+    WHERE r.project_id = '{project_id}'
+    AND rt.id IS NULL
+
     """
+    if query_only:
+        return query
+    if limit > 0:
```

Comment:
the default value is 0, however, the function is also used with None. This will still have the same result however < 0 would result in some [strange behavior on None types ](https://stackoverflow.com/a/7383866/19801189) (wouldn't make sense context-wise but to prevent copy-paste issues I would check for none or ensure the function isn't used with None)

Reply:
set a default of 0 for limit